src/aerie_cli/commands/scheduling.py

```diff
@@ -8,9 +8,9 @@
 def upload(
     model_id: int = typer.Option(
         ..., help="The mission model ID to associate with the scheduling goal", prompt=True
-    ),
+    ),   
     plan_id: int = typer.Option(
-        ..., help="Plan ID", prompt=True
+        help="Plan ID (optional)", default=-1, prompt=True #how to make this optional
```

Comment:
Should the user still be prompted for the plan-id if it's now optional? 

Right now, I still have it prompting the user for the plan-id, but if the user does not input anything and skips past it, then the default value is -1 (not a valid plan-id).

Reply:
You'll need to set `prompt` to `False`. IMO, the intuitive interface here would be to specify *either* the Plan ID or a Model ID, but not both, and throw an error if the user specifies both. If you want to preserve some sort of prompts, you can manually implement the logic with a menu (i.e., prompt the user to select whether they upload to a single plan or all plans for a given model). I have a [utility](https://github.com/NASA-AMMOS/aerie-cli/blob/80159edfff1d085bc8442dead2f3b08d6b6bd2ee/src/aerie_cli/utils/prompts.py#L6-L14) that should make this easier.

Reply:
It looks like the Model ID is used within the the scheduling upload object, so I'm not sure if model id still needs to be an input when there is also a plan id.  I could also query the model ID from the plan ID in order to assert that both are not inputted. 
I will still make a selection menu so the user knows the two options and set both model id and plan id prompts to False.

Reply:
That's a good point -- yeah, you should be able to get the model ID from the plan ID (using the same `list_all_activity_plans` method as below, if you want).

Reply:
I decided to not throw an error when both model id and plan id are inputted. I added a check if the model id is not inputted, then I find the model id from `list_all_activity_plans`. 
I’m also making a couple checks to ensure the inputs that are needed are there using the `select_from_list` method. I set the schedule prompt to False to keep it consistent with the other inputs so they all inputs are prompted together.